Sprite/Sprite.swift

```diff
@@ -87,14 +86,14 @@ public class Sprite: UIView {
   private var _animated: Bool = false
   
   public func animate( shouldPlay: Bool ) {
-    setAnimated(shouldPlay)
+    setAnimated(shouldPlay: shouldPlay)
   }
   
   // bridge for React property setter
   public func setAnimated( shouldPlay:Bool ) {
     _animated = shouldPlay
     
-    if _animated != sprite!.isAnimating() {
+    if _animated != sprite!.isAnimating {
       if shouldPlay {
         sprite!.animationImages = seq
         sprite!.animationRepeatCount = _repeatCount
```

Comment:
This still uses the `_repeatCount`, which you removed above in `setRepeatCount()`. Can we remove this line safely?